synapse/rest/client/room_batch.py

```diff
@@ -213,7 +214,7 @@ async def on_POST(
         prev_state_ids = list(prev_state_map.values())
         auth_event_ids = prev_state_ids
 
-        state_events_at_start = []
+        state_event_ids_at_start = []
```

Comment:
unrelated to this PR, but this method really needs splitting up. I shouldn't have to scroll through over 200 lines to find out where a variable is used!

Reply:
👍 Definitely. I've been holding off on refactors like this while so many PRs have been in flight 😬